src/components/SettingsDialog/Participants/Participants.scss

```diff
@@ -266,6 +275,39 @@
   }
 }
 
+.participant__join-icon,
+.participant__kick-icon {
+  width: $icon--large;
+  height: $icon--large;
+  padding: 0;
+  margin-left: auto; // align to the very right
+  border-radius: $note__border-radius;
```

Comment:
Could be nitpicking, but I'd prefer if you use our new border radius constants `$rounded--*`